harness/determined/deploy/gcp/cli.py

```diff
@@ -185,6 +266,19 @@ def parse(s: str) -> Tuple[str, str]:
                     "optional named arguments",
                     None,
                     [
+                        Arg(
+                            "--cluster-id",
+                            type=str,
+                            default=os.getcwd(),
+                            help="cluster id of the cluster to be deleted",
+                        ),
+                        Arg(
+                            "--project-id",
+                            type=str,
+                            default=None,
+                            required=True,
+                            help="project ID that the cluster belongs to",
```

Comment:
we are a bit inconsistent on the `ID` capitalization. I'd prefer to have it lowercase everywhere.

Reply:
Done